### PR TITLE
Integration2 t6f2 91 reviews refurbish

### PR DESCRIPTION
**Pull request by bovornc**
Minor tweaks to work with reviews backend

### CODE REVIEW

File: src/services/reviews-service.js [hunk 1]
```diff
@@ -2,25 +2,32 @@ import axios from "axios";
 const BASE_URL = "http://localhost:4000";
 
 const RESTAURANTS_API = `${BASE_URL}/api/restaurant`;
+const REVIEWS_API = `${BASE_URL}/api/reviews`;
```
Comment (nala-he) [inline src/services/reviews-service.js:6]: Just a note that we need to add  the following line to enable the user signup/loggin with credentials here. But we will not need this line now.

const api = axios.create({
    withCredentials: true
});

**Review by nala-he — APPROVED** [review]
Looks good to me! I think you will need to refactor some codes in the review component of the react project this week to utilize this reviews-service and check if the backend and frontend actually connect and work. We cannot tell that now as this service is not currently used in the front end.

File: src/services/reviews-service.js [hunk 2]
```diff
@@ -1,26 +1,35 @@
...
+        .then(response => response.data)
+
+export const findAllReviewsForRestaurant = (restaurantid) =>
     axios.get(`${RESTAURANTS_API}/${restaurantid}/reviews`)
         .then(response => response.data);
 
-export const createReview = (restaurantid, uid, review) =>
-    axios.post(`${RESTAURANTS_API}/${restaurantid}/users/${uid}/reviews`, review)
+export const findAllReviewsByCritic = (criticid) =>
+    axios.get(`${BASE_URL}/users/${criticid}/reviews`)
```
Comment (MaceeQi) [inline src/services/reviews-service.js]: will need to add api here for this to work since BASE_URL doesn't include api

**Review by MaceeQi — COMMENTED** [review]
Also agree with Yutong, can't fully test whether all the services work until we try it out with the front end, but based on the URLs, findAllReviewsByCritic will need to add api since BASE_URL does not include it. The rest of the get, as well as create, update, and delete worked well on Postman from the Node project and url matches up so I don't see any problems there as of now

**Review by MaceeQi — APPROVED** [review]
Thanks for the changes!

**Pull request merged**